src/main/java/com/google/firebase/remoteconfig/RemoteConfigParameter.java

```diff
@@ -122,9 +136,21 @@ ParameterResponse toParameterResponse() {
     for (Map.Entry<String, RemoteConfigParameterValue> entry : conditionalValues.entrySet()) {
       conditionalResponseValues.put(entry.getKey(), entry.getValue().toParameterValueResponse());
     }
-    ParameterValueResponse parameterValueResponse = (defaultValue == null) ? null : defaultValue
-            .toParameterValueResponse();
-    return new ParameterResponse(parameterValueResponse, description,
-            conditionalResponseValues);
+    ParameterValueResponse defaultValueResponse = (defaultValue == null) ? null
+            : defaultValue.toParameterValueResponse();
+    return new ParameterResponse()
+            .setDefaultValue(defaultValueResponse)
+            .setDescription(description)
+            .setConditionalValues(conditionalResponseValues);
+  }
+
+  private RemoteConfigParameterValue fromParameterValueResponse(
```

Comment:
I think this ought to be in the `ParameterValue` class. `ParameterValue.fromParameterValueResponse()` makes sense.